rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java

```diff
@@ -74,7 +73,7 @@ void setUp() {
     @TestInstance(TestInstance.Lifecycle.PER_CLASS)
     @Tag("null federations")
```

Comment:
It would be great if we could avoid whitespaces in tags. We can use a camel case.